db/db.go

```diff
@@ -1,115 +1,114 @@
 package db
 
 import (
+	"context"
 	"errors"
-	"github.com/jinzhu/gorm"
-	gormbulk "github.com/t-tiger/gorm-bulk-insert"
 	"github.com/trustwallet/blockatlas/db/models"
 	"github.com/trustwallet/blockatlas/pkg/logger"
-	"go.elastic.co/apm/module/apmgorm"
-	_ "go.elastic.co/apm/module/apmgorm/dialects/postgres"
+	"gorm.io/driver/postgres"
+	"gorm.io/gorm"
+	gormlogger "gorm.io/gorm/logger"
+	"log"
+	"os"
 	"reflect"
 	"time"
 )
 
 type Instance struct {
-	Gorm     *gorm.DB
-	GormRead *gorm.DB
+	Gorm *gorm.DB
 }
 
 const batchCount = 1000
 
-func New(uri, readURI, env string, mode bool) (*Instance, error) {
-	var (
-		g   *gorm.DB
-		rg  *gorm.DB
-		err error
-	)
-	if env == "prod" {
-		g, err = apmgorm.Open("postgres", uri)
-	} else {
-		g, err = gorm.Open("postgres", uri)
-	}
-
-	if err != nil {
-		return nil, err
-	}
-
-	if env == "prod" {
-		rg, err = apmgorm.Open("postgres", readURI)
-	} else {
-		rg, err = gorm.Open("postgres", readURI)
+func New(uri string, logMode bool) (*Instance, error) {
+	cfg := &gorm.Config{}
+	if logMode {
+		cfg.Logger = gormlogger.New(
+			log.New(os.Stdout, "\r\n", log.LstdFlags),
+			gormlogger.Config{
+				SlowThreshold: time.Second,
+				LogLevel:      gormlogger.Info,
+				Colorful:      false,
+			},
+		)
 	}
-
+	g, err := gorm.Open(postgres.Open(uri), cfg)
```

Comment:
```suggestion
	db, err := gorm.Open(postgres.Open(uri), cfg)
```

for consistency let's use `db` as in other places